### DISCUSSION TITLE
Kiosk mode that hides sidebar but shows dashboard variables

### DISCUSSION THREAD

**Discussion by nabelekt** [grafana/grafana#64472] [Dashboards, Panels & Visualizations]
We could use a kiosk mode that hides sidebar but shows dashboard variables. We always need variables shown, so that means that, currently, we also have to have the sidebar shown. If this was implemented, it would be the only mode that we would use in operation.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82830. Feel free to continue the discussion around this there. Thank you!
